src/components/List/List.jsx

```diff
@@ -64,6 +70,55 @@ const List = () => {
 
   const clearFilter = () => setInputValue('');
 
+  const groups = (list) => {
+    const listByGroups = [[], [], [], []];
```

Comment:
I think the role of this matrix is not clear. You can use a dictionary to define a name for each subarray.